crates/sui-graphql-rpc/src/context_data/db_data_provider.rs

```diff
@@ -1609,34 +1611,6 @@ impl TryFrom<StoredObject> for Coin {
     }
 }
 
-impl From<SuiStake> for Stake {
```

Comment:
are we getting rid of this one because this is a duplicate of code elsewhere?

Reply:
This was a leftover from `SuiStake` which is from JSON-RPC type and we don't need it now.